craco.config.js

```diff
@@ -57,24 +55,6 @@ module.exports = function (options) {
           chunkFilename: "static/css/[name].[contenthash:8].chunk.css",
           ignoreOrder: true,
         }),
-        new webpack.NormalModuleReplacementPlugin(
```

Comment:
@mattrussell36 I think the tests are still using these mocks so we'd need to find another solution before merging this. Obviously need to decide how we want to mock these ect.

Reply:
Let me double check this. It looked like the tests weren't using them anymore.